sdks/java/core/src/main/java/org/apache/beam/sdk/schemas/Schema.java

```diff
@@ -474,6 +493,29 @@ public boolean equals(Object o) {
           && Arrays.equals(getMetadata(), other.getMetadata());
     }
 
+    /** Returns true if two FieldTypes are equal, ignoring name and description. */
```

Comment:
It doesn't seem to ignore the type name, copy-paste error?

Reply:
Confusing terminology - "TypeName" is the type, not the field name.